<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AsciiFlow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnToolMove">
        <property name="text">
         <string>Move</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnToolRectangle">
        <property name="text">
         <string>Rectangle</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnToolClass">
        <property name="text">
         <string>Class</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnToolArrow">
        <property name="text">
         <string>Arrow</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnToolText">
        <property name="text">
         <string>Text</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnToolErase">
        <property name="text">
         <string>Erase</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnToolFreehand">
        <property name="text">
         <string>Freehand</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QAsciiArt" name="customArea" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionExport"/>
   <addaction name="actionImport"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Import</string>
   </property>
  </action>
  <action name="actionToolMove">
   <property name="text">
    <string>Move</string>
   </property>
   <property name="toolTip">
    <string>Move</string>
   </property>
  </action>
  <action name="actionToolRectangle">
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="toolTip">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionToolClass">
   <property name="text">
    <string>Class</string>
   </property>
   <property name="toolTip">
    <string>Class</string>
   </property>
  </action>
  <action name="actionToolArrow">
   <property name="text">
    <string>Arrow</string>
   </property>
  </action>
  <action name="actionToolErase">
   <property name="text">
    <string>Erase</string>
   </property>
   <property name="toolTip">
    <string>Erase</string>
   </property>
  </action>
  <action name="actionToolFreehand">
   <property name="text">
    <string>Freehand</string>
   </property>
   <property name="toolTip">
    <string>Freehand</string>
   </property>
  </action>
  <action name="actionToolText">
   <property name="text">
    <string>Text</string>
   </property>
   <property name="toolTip">
    <string>Text</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QAsciiArt</class>
   <extends>QWidget</extends>
   <header>qasciiart.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
